Auto-Instrumentation ERROR $_SERVER variable not populated

**Describe the bug**
APM-Agent version: 1.8.2
Container: php-fpm-alpine
PHP version: 8.1
Relevant Extensions and framework:
- Symfony 6
- opcache: enabled
- Nginx version: latest

After upgrading from `1.8.1` to `1.8.2` In every request I'm getting the next 2 errors:
```
[ERROR] [Auto-Instrumentation] [TransactionForExtensionRequest.php:173] [discoverHttpRequestData] $_SERVER variable is not populated even after forcing PHP engine to populate it - agent will have to fallback on defaults {"namespace":"Elastic\\Apm\\Impl\\AutoInstrument","class":"TransactionForExtensionRequest","this":{"type":"Elastic\\Apm\\Impl\\AutoInstrument\\TransactionForExtensionRequest","object_ID":3,"object_hash":"00000000000000030000000000000000"},"stacktrace":[{"file":"TransactionForExtensionRequest.php","line":176,"class":"TransactionForExtensionRequest","function":"discoverHttpRequestData"},{"file":"TransactionForExtensionRequest.php","line":126,"class":"TransactionForExtensionRequest","function":"beginTransaction"},{"file":"TransactionForExtensionRequest.php","line":89,"class":"TransactionForExtensionRequest","function":"__construct"},{"file":"PhpPartFacade.php","line":79,"class":"PhpPartFacade","function":"__construct"},{"file":"PhpPartFacade.php","line":113,"class":"PhpPartFacade","function":"bootstrap"},[]]}
```

```
[ERROR] [Auto-Instrumentation] [TransactionForExtensionRequest.php:485] [getMandatoryServerVarElement] $_SERVER does not contain `REQUEST_TIME_FLOAT' key {"namespace":"Elastic\\Apm\\Impl\\AutoInstrument","class":"TransactionForExtensionRequest","this":{"type":"Elastic\\Apm\\Impl\\AutoInstrument\\TransactionForExtensionRequest","object_ID":3,"object_hash":"00000000000000030000000000000000"},"stacktrace":[{"file":"TransactionForExtensionRequest.php","line":486,"class":"TransactionForExtensionRequest","function":"getMandatoryServerVarElement"},{"file":"TransactionForExtensionRequest.php","line":556,"class":"TransactionForExtensionRequest","function":"discoverStartTime"},{"file":"TransactionForExtensionRequest.php","line":132,"class":"TransactionForExtensionRequest","function":"beginTransaction"},{"file":"TransactionForExtensionRequest.php","line":89,"class":"TransactionForExtensionRequest","function":"__construct"},{"file":"PhpPartFacade.php","line":79,"class":"PhpPartFacade","function":"__construct"},{"file":"PhpPartFacade.php","line":113,"class":"PhpPartFacade","function":"bootstrap"},[]]}
```

This error is making the traces data sent as `Unnamed transaction` in the APM server.

**To Reproduce**
Steps to reproduce the behavior:
1. Use this config `elastic_apm.server_url = $APM_SERVER_URL
elastic_apm.secret_token = ""
elastic_apm.service_name = "$APP_NAME"
elastic_apm.enabled = $APM_ENABLED
elastic_apm.environment = $APP_ENV
elastic_apm.log_level = WARNING
elastic_apm.service_version = $APP_VERSION
elastic_apm.transaction_sample_rate=0.3'`
2. Then make any http request to the service
4. See error

**Expected behavior**
The error not happening and trace data sent with correct transaction name

--- a/src/ext/lifecycle.c
+++ b/src/ext/lifecycle.c
@@ -590,9 +590,9 @@ void elasticApmRequestInit()
     ELASTIC_APM_CALL_IF_FAILED_GOTO( ensureAllComponentsHaveLatestConfig( tracer ) );
     logSupportabilityInfo( logLevel_trace );
 
+    enableAccessToServerGlobal();
 
     if (requestCounter == 1) {
-        enableAccessToServerGlobal();
         bool preloadDetected = detectOpcachePreload();
         if (preloadDetected) {
             ELASTIC_APM_LOG_DEBUG( "opcache.preload request detected on init" );

